### PR TITLE
Main to beta

### PR DESCRIPTION
**Pull request by ErnoW**
---
name: 'Pull request'
about: A new pull request
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/CODE_STYLE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Issue Description

Related issue: #`FILL_THIS_OUT`

### CODE REVIEW

**Comment by ErnoW** [comment]
Still to do:
- [ ] Fix tests
- [ ] Merge main 2.7.4

File: packages/auth/package.json [hunk 1]
```diff
@@ -28,9 +28,9 @@
     "typescript": "^4.5.5"
   },
   "dependencies": {
-    "@moralisweb3/api-utils": "^2.7.1",
-    "@moralisweb3/common-core": "^2.7.1",
-    "@moralisweb3/common-evm-utils": "^2.7.1",
-    "@moralisweb3/common-sol-utils": "^2.7.1"
+    "@moralisweb3/api-utils": "^2.7.4",
+    "@moralisweb3/common-core": "^2.7.4",
+    "@moralisweb3/evm-utils": "^2.7.4",
```
Comment (sogunshola) [inline packages/auth/package.json:33]:
```suggestion
    "@moralisweb3/common-evm-utils": "^2.7.4",
```

File: packages/auth/package.json [hunk 2]
```diff
@@ -28,9 +28,9 @@
...
+    "@moralisweb3/sol-utils": "^2.7.4"
```
Comment (sogunshola) [inline packages/auth/package.json:34]:
```suggestion
    "@moralisweb3/common-sol-utils": "^2.7.4"
```

**Review by sogunshola — CHANGES_REQUESTED** [review]

File: packages/moralis/README.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+  <p>
+  </p>
+  <p>
+    A library that gives you access to the powerful Moralis Server backend from your JavaScript app.
+  </p>
+  <br/>
+</div>
+
+---
+
```
Comment (b4rtaz) [inline packages/moralis/README.md:28]: This file should be not added to the repo. It automatically updates from the root (I added ` "prebuild": "cp -f ../../README.md README.md"` lately).

**Review by sogunshola — DISMISSED** [review]
Looks good from my eyes.

File: packages/common/solUtils/src/operations/operations.test.ts [hunk 4]
```diff
@@ -2,6 +2,11 @@ import { toCamel } from '@moralisweb3/common-core';
 import { OpenApiInterfaceReader, OperationDefinitionReader } from '@moralisweb3/test-utils';
 import { operations } from './operations';
 
+const ignoreUrlSearchCheckOperationNames = [
+  // Test fails when having multiple return statements due to validation check
+  'getTokenPrice',
+];
```
Comment (b4rtaz) [inline packages/common/solUtils/src/operations/operations.test.ts:8]: We can fix it easily. In the `packages/testUtils/src/operations/OperationDefinitionReader.ts` file, the `getRequestUrlParamsFunctionReturnPropertyNames` method should find **any first return statement**. 

``` 
    const func = funcs[0]; // This should be replaced by searching the first return statement.
```

**Review by b4rtaz — CHANGES_REQUESTED** [review]

**Review by b4rtaz — APPROVED** [review]

**Pull request merged**